registry.json

```diff
@@ -1,4 +1,20 @@
 [
+  {
+    "ticker": "MBX",
+    "poolId": "26418769855d77d899fd00923bfbb0a9052a843bc0b0f7a5a94bbc73",
+    "social": {
+      "website": "MaineBlockchain.com",
+      "discord": "discord.gg/VSzKnk8xcq",
+      "reddit": "MaineResident",
+      "instagram": "MaineBlockchain",
+      "facebook": "https://www.facebook.com/defistrategies",
+      "youtube": "BlackBearCrypto",
+      "twitter": "MaineBlockchain"
+    },
+    "telegram": "MBX_DeFiStrategies",
```

Comment:
I couldnt find this telegram handle... please can you check it is correct and that it is the handle of a user and not a group or channel?